sql/migrations/0001_initial.sql

```diff
@@ -17,6 +17,12 @@ CREATE TABLE ballots
 );
 CREATE INDEX ballots_by_layer_by_pubkey ON ballots (layer, pubkey);
 
+CREATE TABLE ref_ballots
```

Comment:
why does ref ballot need to be a separate table and not a field of ballots table?

btw, the primary key should be (epoch, smesher public key)